ctedd-export.iml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You'll probably need to close IntelliJ, delete these IntelliJ files that shouldn't be under version control, commit that from the command-line, and then re-import into IntelliJ to get rid of these files in version control.